components/style/values/computed/font.rs

```diff
@@ -94,3 +98,36 @@ impl ToAnimatedValue for FontSize {
         }
     }
 }
+
+impl ToComputedValue for specified::MozScriptMinSize {
+    type ComputedValue = MozScriptMinSize;
+
+    fn to_computed_value(&self, cx: &Context) -> MozScriptMinSize {
+        // this value is used in the computation of font-size, so
+        // we use the parent size
+        let base_size = FontBaseSize::InheritedStyle;
+        match self.0 {
+            NoCalcLength::FontRelative(value) => {
+                value.to_computed_value(cx, base_size)
+            }
+            NoCalcLength::ServoCharacterWidth(value) => {
+                value.to_computed_value(base_size.resolve(cx))
+            }
+            ref l => {
+                l.to_computed_value(cx)
+            }
+        }
+    }
+
+    fn from_computed_value(other: &MozScriptMinSize) -> Self {
+        specified::MozScriptMinSize(ToComputedValue::from_computed_value(other))
+    }
+}
+
+impl Parse for specified::MozScriptMinSize {
```

Comment:
nit: `Parse` implementations go usually in the `specified` module, mind moving this there?

Also, there's some weird indentation below (not really your fault), mind fixing it up like:

```rust
    fn parse<'i, 't>(_: &ParserContext, input: &mut Parser<'i, 't>) -> Result<Self, ParseError<'i>> {
        // ...
    }
```

With that, this looks great, thanks a lot!

Reply:
Let me move it into `specified` module.